samples/rust/src/wc_proto/README.md

```diff
@@ -0,0 +1,2 @@
+Source files generated from the protobuf .proto files will come here, during build time.
```

Comment:
I think you can create this folder during cargo build? no need to add a placeholder

Reply:
+1